client/my-sites/plugins/plugin-automated-transfer/index.jsx

```diff
@@ -40,18 +43,22 @@ class PluginAutomatedTransfer extends Component {
 	};
 
 	componentWillMount() {
-		const { COMPLETE, CONFLICTS } = transferStates;
-		const { isTransferring, transferState } = this.props;
+		const { COMPLETE } = transferStates;
+		const {
+			isTransferring,
+			isTransferFailed,
+			transferState,
+		} = this.props;
 
 		if ( COMPLETE === transferState ) {
 			this.setState( { transferComplete: true } );
-		} else if ( isTransferring || CONFLICTS === transferState ) {
+		} else if ( isTransferring || isTransferFailed ) {
```

Comment:
the naming here should probably change for grammar purposes: `didTransferFail` `hasTransferFailed` or `isFailedTransfer` might all be more fitting.